# HG changeset patch
# User Rik <rik@octave.org>
# Date 1397841086 25200
#      Fri Apr 18 10:11:26 2014 -0700
# Node ID cce0a63afb3c1fddb64674bddb1d972adc45a754
# Parent  4f43f87b7c3ef0c405912764ac0da9832292e4cd
# Parent  75ec138ba53b6c92284866821f41a31a0b977169
maint: Merge away extra head.

diff --git a/libinterp/corefcn/gl-render.cc b/libinterp/corefcn/gl-render.cc
--- a/libinterp/corefcn/gl-render.cc
+++ b/libinterp/corefcn/gl-render.cc
@@ -1543,17 +1543,17 @@ opengl_renderer::draw_line (const line::
 
       init_marker (props.get_marker (), props.get_markersize (),
                    props.get_linewidth ());
 
       for (int i = 0; i < n; i++)
         {
           if (clip[i] == clip_ok)
             draw_marker (x(i), y(i),
-                         has_z ? z(i) : static_cast<double> (i) / n,
+                         has_z ? z(i) : 0.0,
                          lc, fc);
         }
 
       end_marker ();
     }
 
   set_clipping (props.is_clipping ());
 }
